iset.mm

```diff
@@ -117398,6 +117398,32 @@ Infinite sums (cont.)
       WDIXBKWEMWATTOSWGWHNRQZMJZIJWJVPWCXEWIWGIXDUBWHMWBTTOS $.
   $}
 
+  ${
+    $d A k n y $.
+    $( The exponential function of a positive real number is greater than the
```

Comment:
The exponential of... the **sum of** the first three...

Reply:
Changed in both set.mm and iset.mm